{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -1,3 +1,62 @@
-# Model Monitoring
+# Setup Data Monitoring for Inference Table
+
+**Databricks Data Monitoring is currently in Private Preview. Please `sign up` for the preview and get approved
```

Comment:
Any way we can provide more information or a link on how to sign up?

Reply:
We're discussing within our team about whether we want to expose our questionnaire to the public internet. Will update y'all when we have an answer - in the mean time, maybe we can say something like "Please reach out to your Databricks representative in order to sign up for the preview"?